### ISSUE TITLE
:json: dependencies result in a jvm_import rule failure

### ISSUE THREAD

**Issue by mmikitka** [bazelbuild/rules_jvm_external#351]
Using version 3.1 of rules_jvm_external and Bazel version 1.1.0

Here is my maven_install statement.
```
QUARKUS_VERSION = "1.0.1.Final"
maven_install(
        name = "maven",
        artifacts = [
            "io.quarkus:quarkus-bom-descriptor-json:" + QUARKUS_VERSION,
        ],
        repositories = [
            "https://repo.maven.apache.org/maven2/",
            "https://repo.spring.io/plugins-release/",
        ],
        fetch_sources = True,
        maven_install_json = "//:maven_install.json",
    )
```

Running the unpinned command generates the following error:

```
bazel run @unpinned_maven//:pin

ERROR: An error occurred during the fetch of repository 'maven':
   Traceback (most recent call last):
	File "/home/mmikitka/.cache/dazel/_dazel_mmikitka/021e3d924d9514e8b95df637aaba1c98/external/rules_jvm_external/coursier.bzl", line 316
		parser.generate_imports(<6 more arguments>)
	File "/home/mmikitka/.cache/dazel/_dazel_mmikitka/021e3d924d9514e8b95df637aaba1c98/external/rules_jvm_external/private/dependency_tree_parser.bzl", line 121, in parser.generate_imports
		fail(<1 more arguments>)
Unsupported packaging type: json
ERROR: no such package '@maven//': Traceback (most recent call last):
	File "/home/mmikitka/.cache/dazel/_dazel_mmikitka/021e3d924d9514e8b95df637aaba1c98/external/rules_jvm_external/coursier.bzl", line 316
		parser.generate_imports(<6 more arguments>)
	File "/home/mmikitka/.cache/dazel/_dazel_mmikitka/021e3d924d9514e8b95df637aaba1c98/external/rules_jvm_external/private/dependency_tree_parser.bzl", line 121, in parser.generate_imports
		fail(<1 more arguments>)
Unsupported packaging type: json
ERROR: no such package '@maven//': Traceback (most recent call last):
	File "/home/mmikitka/.cache/dazel/_dazel_mmikitka/021e3d924d9514e8b95df637aaba1c98/external/rules_jvm_external/coursier.bzl", line 316
		parser.generate_imports(<6 more arguments>)
	File "/home/mmikitka/.cache/dazel/_dazel_mmikitka/021e3d924d9514e8b95df637aaba1c98/external/rules_jvm_external/private/dependency_tree_parser.bzl", line 121, in parser.generate_imports
		fail(<1 more arguments>)
Unsupported packaging type: json

```

**Comment by mmikitka** [comment]
A related conversation in the Coursier repo: https://github.com/coursier/coursier/issues/1465

**Comment by jin** [comment]
Fixed by #352
